config/routes.rb

```diff
@@ -3,8 +3,9 @@
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
- root 'dashboard#index'
-
+ root 'urls#index'
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.